andela/denethor-ah-backend
# #163782038 Implement user session management with cookies

#### What does this PR do?
 This pull request implements session management with cookies.

#### Description of Task to be completed?
- Set up cookies express-sessions
- Set up PostgreSQL session store with connect-pg-simple

#### How should this be manually tested?
- Clone the repo
- Checkout branch `feature/163782038/session-management`
- Run `npm i` to install dependencies
- Set up `.env` file as specified in the `.env.sample file`, set the port to 5000
- Run migrations with `npm run migrations`
- Start server with `npm run start:dev`
- Open your browser and visit `http://localhost:5000/api/users/google` to login with google
- After logging in check your sessions table in your database to see session storage for your login session

#### What are the relevant pivotal tracker stories?
[#163782038](https://www.pivotaltracker.com/story/show/163782038)
